lib/media/streaming_engine.js

```diff
@@ -1066,7 +1072,13 @@ shaka.media.StreamingEngine = class {
       recovering: false,
       hasError: false,
       operation: null,
-    });
+      dependencyMediaState: null,
+    };
+    if (stream.dependencyStream) {
+      mediaState.dependencyMediaState =
+          this.createMediaState_(stream.dependencyStream);
+    }
+    return /** @type {!shaka.media.StreamingEngine.MediaState_} */(mediaState);
```

Comment:
I think it will be cleaner to add type to variable definition instead of casting it here.

Reply:
Done!